wallet/storage/src/internal/store_tx.rs

```diff
@@ -63,13 +67,68 @@ macro_rules! impl_read_ops {
                 self.read_value::<well_known::StoreVersion>().map(|v| v.unwrap_or_default())
             }
 
-            fn get_utxo(&self, outpoint: &OutPoint) -> crate::Result<Option<Utxo>> {
+            fn get_utxo(&self, outpoint: &AccountOutPointId) -> crate::Result<Option<Utxo>> {
                 self.read::<db::DBUtxo, _, _>(outpoint)
             }
 
-            fn get_transaction(&self, id: &Id<Transaction>) -> crate::Result<Option<WalletTx>> {
+            /// Collect and return all utxos from the storage
+            fn read_utxo_set(
```

Comment:
Such functions can cause huge performance issues. I know wallets that have millions of UTXOs...